python/lsst/ts/externalscripts/auxtel/latiss_cwfs_align.py

```diff
@@ -432,33 +374,42 @@ async def run_cwfs(self):
                 f"Running cwfs in " f"{self.intra_visit_id}/{self.extra_visit_id}."
             )
 
-        self.intra_exposure = await loop.run_in_executor(
-            executor, self.get_isr_exposure, self.intra_visit_id
+        self.intra_exposure, self.extra_exposure = await asyncio.gather(
+            get_image(parse_visit_id(self.intra_visit_id)),
+            get_image(parse_visit_id(self.extra_visit_id)),
         )
 
-        # create a clone to be used for detection
-        self.detection_exp = self.intra_exposure.clone()
+        # Run source detection - FIXME: make this run concurrently
```

Comment:
You are already logging "Run source detection" I think you can leave just the "FIXME" part.

Reply:
Actually removed the FIXME because we decided running concurrently was not worth it.